packages/components/src/internal/components/domainproperties/samples/SampleTypePropertiesPanel.tsx

```diff
@@ -338,6 +356,29 @@ class SampleTypePropertiesPanelImpl extends React.PureComponent<
                         </Col>
                     </Row>
                 )}
+
+                {allowTimepointProperties && !appPropertiesOnly && (
+                    <Row className="margin-top">
+                        <Col xs={2}> Auto-Link Data to Study </Col>
+                        <Col xs={5}>
+                            <FormControl
+                                componentClass="select"
+                                id={ENTITY_FORM_IDS.AUTO_LINK_TARGET}
+                                onChange={this.onFormChange}
+                                value={model.autoLinkTargetContainerId || ''}
+                            >
+                                <option key="_empty" value={null} />
+                                {containers &&
+                                    containers.map((container, i) => (
+                                        <option key={i} value={container.id}>
```

Comment:
The `AssayPropertiesInput` case has the key as the `i` index, but really it should be something other then the loop index ideally. I think in this case using `container.id` for the key in both cases would make sense.

Reply:
Updated across both!